src/commands/init/generateTemplate/index.js

```diff
@@ -73,7 +73,7 @@ function filterFiles(filters) {
 function render(template, data, callback) {
     let rendered;
     try {
-        rendered = Handlebars.compile(template)(data);
+        rendered = Handlebars.compile(template, data)(data);
```

Comment:
I think we always want to use `noEscape` and it was a bug that this was not done before.